src/open-websocket.js

```diff
@@ -0,0 +1,14 @@
+import Html5WebSocket from 'html5-websocket'
+import ReconnectingWebSocket from 'reconnecting-websocket'
+import WebSocket from 'ws'
+
+export default wsAddressBuilder => {
+  return new ReconnectingWebSocket(wsAddressBuilder, undefined, {
+    connectionTimeout: 4000,
+    constructor: typeof window !== 'undefined' ? WebSocket : Html5WebSocket,
+    debug: false,
+    maxReconnectionDelay: 10000,
+    maxRetries: Infinity,
```

Comment:
Let's put this one to `100` (ps: the lib accepts `null` to represent infinity)

Reply:
Why do you want to limit `maxRetries`? In case of a longer downtime of the Binance API (which happened already a lot in the past), it's good to retry as much as we can. I can remove the entry because `Infinity` is already the default but I wanted to be explicit so that you can see which values can be tweaked.

Reply:
Okay let's keep it that way